src/main/steps/applicant1/finalising-your-application/content.ts

```diff
@@ -1,18 +1,23 @@
 import { getFormattedDate } from '../../../app/case/answers/formatDate';
 import { Checkbox } from '../../../app/case/case';
+import { State } from '../../../app/case/definition';
 import { TranslationFn } from '../../../app/controller/GetController';
 import { FormContent } from '../../../app/form/Form';
 import { isFieldFilledIn } from '../../../app/form/validation';
 import { CommonContent } from '../../common/common.content';
 import { generateContent as columnGenerateContent } from '../hub-page/right-column/content';
 
-const en = ({ isDivorce, partner, userCase }: CommonContent) => ({
+const en = ({ isDivorce, partner, userCase, isJointApplication }: CommonContent) => ({
   title: `Do you want to ${isDivorce ? 'finalise your divorce' : 'end your civil partnership'}?`,
   line1: `Your ${
     isDivorce ? 'marriage' : 'civil partnership'
   } will be legally ended after the final order is made. This might affect your finances.`,
   warningText: `If you have not finished negotiations about your money, property or other assets then you should seek legal advice before finalising
-  ${isDivorce ? 'your divorce' : 'ending your civil partnership'}`,
+  ${
+    isDivorce || State.AwaitingJointFinalOrder.includes(userCase.state as State)
+      ? 'your divorce'
+      : 'ending your civil partnership'
```

Comment:
won't this mean that civil partnership cases in state AwaitingJointFinalOrder will see 'your divorce'